### DISCUSSION TITLE
When playing a picture (title cover) should be displayed

### DISCUSSION THREAD

**Discussion by Lehmeiers** [enzo1982/freac#316] [Your Ideas]
Good day!
I use Fre:ac under Manjaro and it is very impressive.
It is really class, but I am missing a way to embed an image so that it is displayed when playing. I do not find such a possibility.

Is there none and will you still implement such a possibility?

Sincerly R.Lehmeier

**Comment by enzo1982** [comment]
Hi, you can add or remove cover images on the _Tags_ tab in fre:ac. Either for each track individually or for all tracks belonging to an album at once. See this screenshot:

![grafik](https://user-images.githubusercontent.com/7796713/169500280-cacbb16e-51cd-41d2-b24c-4cdf7a29d00d.png)
